packages/yourloops/pages/hcp/team-members-table.tsx

```diff
@@ -39,7 +39,6 @@ import Avatar from "@material-ui/core/Avatar";
 import Checkbox from "@material-ui/core/Checkbox";
```

Comment:
Maybe changing this file was a little out of scope ?
There is enough changed already I think, it can be done in another PR.